code/backend/batch/utilities/helpers/ConfigHelper.py

```diff
@@ -70,28 +81,64 @@ def __init__(self, logging: dict):
 
 
 class ConfigHelper:
+    @staticmethod
+    def _set_new_config_properties(config: dict, default_config: dict):
+        """
+        Function used to set newer properties that will not be present in older configs.
+        The function mutates the config object.
+        """
+        if config["prompts"].get("answering_system_prompt") is None:
+            config["prompts"]["answering_system_prompt"] = default_config["prompts"][
+                "answering_system_prompt"
+            ]
+
+        prompt_modified = (
+            config["prompts"].get("answering_prompt")
+            != default_config["prompts"]["answering_prompt"]
+        )
+
+        if config["prompts"].get("answering_user_prompt") is None:
+            if prompt_modified:
+                config["prompts"]["answering_user_prompt"] = config["prompts"][
+                    "answering_prompt"
+                ]
+            else:
+                config["prompts"]["answering_user_prompt"] = default_config["prompts"][
+                    "answering_user_prompt"
+                ]
+
+        if config["prompts"].get("use_new_prompt_format") is None:
+            config["prompts"]["use_new_prompt_format"] = not prompt_modified
+
+        if config.get("example") is None:
+            config["example"] = default_config["example"]
+
     @staticmethod
     def get_active_config_or_default():
         env_helper = EnvHelper()
         config = ConfigHelper.get_default_config()
 
         if env_helper.LOAD_CONFIG_FROM_BLOB_STORAGE:
-            try:
-                blob_client = AzureBlobStorageClient(
-                    container_name=CONFIG_CONTAINER_NAME
-                )
-                config_file = blob_client.download_file("active.json")
-                config = Config(json.loads(config_file))
-            except Exception:
+            blob_client = AzureBlobStorageClient(container_name=CONFIG_CONTAINER_NAME)
+
```

Comment:
This should speed things up quite a bit - many thanks (feel free to resolve once read!) :-)